### PR TITLE
Version 0.2.3

### PR DESCRIPTION
**Pull request by willkg**
This fixes the MANIFEST.in file and updates the project for a 0.2.3 release.

r?

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,4 +1,7 @@
 include README.md
 include LICENSE
+include HISTORY.rst
 include requirements.txt
+
 recursive-include socorrolib *.py
+recursive-include socorrolib *.tst
```
Comment (adngdb) [inline MANIFEST.in:7]: What's that? I don't see any .tst file in this repo. Is it general good practice?

**Comment by adngdb** [comment]
[Spoiler Alert] In today's episode of "The Socorroffice", young Adrian will learn about `.tst` files and general python packaging good practices. Then, with a deep sense of satisfaction for having learned from his senior, Adrian will humbly give Will a well-deserved r+. The episode will end with our protagonists sacrificing a chicken to the Gods for the release 0.2.3 of `socorrolib`.

**Comment by willkg** [comment]
Oops--I totally missed the r+. Landing this now.

**Comment by willkg** [comment]
Thank you!

**Pull request merged**